tozid/sso/oidc/implicit-flow/public/javascripts/main.js

```diff
@@ -6,8 +6,8 @@ Oidc.Log.level = Oidc.Log.INFO;
 //
 // OIDC Client Configuration
 //
-const TOZID_REALM_NAME = 'example'
-const TOZID_CLIENT_ID = 'demo';
+const TOZID_REALM_NAME = 'ssorbacdemo'
```

Comment:
prefer keeping this generic and in sync with both sets of demos. Perfectly fine to copy pasta this whole current app into another folder e.g. `tozid/sso/rbac` and revert this example.